agenttest.py

```diff
@@ -358,6 +357,7 @@ def end_to_end_test():
                  ' --key {cert_dir}/fakeserver.key'
                  ' --gnmiport {gnmi_port}'
                  ' --configfile {cfile}'
+                 ' --nohup {nohup}'
                  ' --dpwait 1.0').format(**params).split()
```

Comment:
We can use an f-string here also I think.

Reply:
@lantz it was already as a `.format`, I was trying to minimize re-writing things for the purposes of getting this feature in. would you prefer it be rewritten for this?